ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageView.java

```diff
@@ -261,10 +291,10 @@ public void maybeUpdateView() {
     boolean usePostprocessorScaling =
         mScaleType != ScalingUtils.ScaleType.CENTER_CROP &&
         mScaleType != ScalingUtils.ScaleType.FOCUS_CROP;
-    float hierarchyRadius = usePostprocessorScaling ? 0 : mBorderRadius;
 
     RoundingParams roundingParams = hierarchy.getRoundingParams();
-    roundingParams.setCornersRadius(hierarchyRadius);
+    roundingParams.setCornersRadii(mBorderRadius);
```

Comment:
You can perhaps use `setCornersRadii(float topLeft, float topRight, float bottomRight, float bottomLeft)` and this way have `mBorderRadius` to only store 4 floats